osf/external/cedar/client.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from urllib.parse import quote_plus
+from website import settings
+
+class CedarClient(object):
+    host = settings.CEDAR_API_HOST
+    api_key = settings.CEDAR_API_KEY
+    home_folder_id = quote_plus(settings.CEDAR_HOME_FOLDER_ID)
+    headers = {
+        'Authorization': f'apikey {api_key}'
+    }
+
+    def retrieve_all_template_ids(self):
+        url = f'{self.host}folders/{self.home_folder_id}/contents/?resource_types=template'
+        r = requests.get(url, headers=self.headers)
+        resources = r.json()['resources']
```

Comment:
We probably need error handling for making the requests and parsing the response.